metricbeat/module/kafka/partition/_meta/fields.yml

```diff
@@ -0,0 +1,38 @@
+- name: partition
+  type: group
+  description: >
+    partition
+  fields:
+    - name: brocker
+      type: integer
+      description: >
+        Broker id
+    - name: offset
+      type: group
+      description: >
+        Available offsets of the given partition.
+      fields:
+        - name: newest
+          type: long
+          description: >
+            Newest offset of the partition.
+        - name: oldest
+          type: long
+          description: >
+            Oldest offset of the partition.
+    - name: partition
+      type: long
+      description: >
+        Partition id.
+    - name: topic
+      type: keyword
+      description: >
+        Topic name
+    - name: replicas
+      type: list
+      description: >
+        List of replicas ids
```

Comment:
Changed it. I saw that there are some differences with notes and was thinking about adding a "status" field but not for the first version.